src/woo_publications/publications/api/serializers.py

```diff
@@ -289,6 +294,7 @@ class Meta:  # pyright: ignore
             "laatst_gewijzigd_datum": {
                 "read_only": True,
             },
+            "bron_bewaartermijn": {"required": True},
```

Comment:
This technically requires us to bump the API version to v2 because it's a breaking change.